x/mongo/driver/description/server.go

```diff
@@ -390,3 +390,9 @@ func decodeStringMap(element bsoncore.Element, name string) (map[string]string,
 	}
 	return m, nil
 }
+
+// SupportsRetryWrites returns true if retryable writes are supported which requires
+// that logicalSessionTimeoutMinutes exists, and server type is not standalone
+func (s *Server) SupportsRetryWrites() bool {
```

Comment:
All of the other methods for `description.Server` use `Server` as the receiver, not `*Server`, so we should stay consistent with that.